docs/source/developer/contributing.rst

```diff
@@ -86,9 +86,17 @@ the label ``foo`` and ``bar baz`` to an issue, comment
 
 Contributing from within the browser
 ------------------------------------
-Using the https://github.com web interface - documented
-`here <https://docs.github.com/en/free-pro-team@latest/github>`__ - you
-can create and propose a change purely within your browser.
+
+Contributing to JupyterLab codebase is also possible without setting up
+a local environment, directly from the Web browser:
+
+-  Gitpod integration is enabled, however it is not actively maintained,
+-  GitHub's built-in editor is suitable for contributing very small fixes,
```

Comment:
I removed the old link as it redirects to main GitHub documentation website (I guess it used to showcase editing or at least link to the relevant section). We could point to the relevant section:

```suggestion
-  GitHub's
   `built-in editor <https://docs.github.com/en/repositories/working-with-files/managing-files/editing-files>`__
   is suitable for contributing very small fixes,
```

but it might get outdated at some point too, so not sure if its worth it.

Reply:
> but it might get outdated at some point too, so not sure if its worth it.

Hopefully the check links CI check should catch the link if it becomes dead in the future?